packages/arb-token-bridge-ui/src/styles/tailwind.css

```diff
@@ -119,6 +119,26 @@ body > iframe {
   border: 1px solid #777;
 }
 
+.tippy-box[data-theme~='dark'] .tippy-arrow:before {
```

Comment:
We have a custom border around dark themed tooltips and it doesn't wrap around the arrow. We need to make our own arrow to make it work.